### PR TITLE
TKNECO-84: Successfully incorporated image-scripts-url parametera

### PR DESCRIPTION
**Pull request by Senjuti256**
With reference to issue #20 issue I could incorporate the image-scripts-url parameter and it is working successfully with the value of image-scripts-url parameter getting parsed successfully during e2e testing.

### CODE REVIEW

File: scripts/s2i-common.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # target image name (fully qualified) to be build with s2i, redeclaring the same parameter name than
 # buildah task uses
 declare -x PARAMS_IMAGE="${PARAMS_IMAGE:-}"
+declare -rx PARAMS_IMAGE_SCRIPTS_URL="${PARAMS_IMAGE_SCRIPTS_URL:-}"
```
Comment (otaviof) [inline scripts/s2i-common.sh:6]: Please leave a top comment with the parameter description.

File: scripts/s2i-generate.sh [hunk 2]
```diff
@@ -13,6 +13,8 @@ source "$(dirname ${BASH_SOURCE[0]})/s2i-common.sh"
 # s2i builder image name (fully qualified)
 declare -rx S2I_BUILDER_IMAGE="${S2I_BUILDER_IMAGE:-}"
 
+declare -rx IMAGE_SCRIPTS_URL="${PARAMS_IMAGE_SCRIPTS_URL:-}"
+
```
Comment (otaviof) [inline scripts/s2i-generate.sh:17]: You can omit this in favor of `PARAMS_IMAGE_SCRIPTS_URL`, declared before.

File: scripts/s2i-generate.sh [hunk 3]
```diff
@@ -51,6 +53,7 @@ fi
 phase "Generating the Containerfile for S2I builder image '${S2I_BUILDER_IMAGE}'"
 s2i --loglevel "${S2I_LOGLEVEL}" \
     build "${PARAMS_SUBDIRECTORY}" "${S2I_BUILDER_IMAGE}" \
+        --image-scripts-url "${IMAGE_SCRIPTS_URL}" \
```
Comment (otaviof) [inline scripts/s2i-generate.sh:56]: Also here, please use `PARAMS_IMAGE_SCRIPTS_URL` variable.

**Review by otaviof — CHANGES_REQUESTED** [review]
Good progress on this 👍 I left a few final comments, please consider.

File: scripts/s2i-generate.sh (hunk 3 above)
Comment (Senjuti256) [reply scripts/s2i-generate.sh]: I have updated the PR as per the review feedback.

**Review by otaviof — APPROVED** [review]
Looks good 👍

**Pull request merged**